include/swift/AST/DiagnosticsParse.def

```diff
@@ -1317,6 +1317,11 @@ ERROR(attr_objc_expected_rparen,none,
       "expected ')' after name for @objc", ())
 ERROR(attr_objc_empty_name,none,
       "expected name within parentheses of @objc attribute", ())
+ERROR(attr_nskeyedarchiverclassname_removed, none,
```

Comment:
Will these errors be removed in the future or do you think this will be here in the code permanently?  I don't think anyone should be using these keyword yet although this does seem like the safest path.